gradio/components.py

```diff
@@ -3930,6 +3930,172 @@ def style(
         return self
 
 
+@document("style")
+class AnnotatedImage(Selectable, IOComponent, JSONSerializable):
```

Comment:
Considering adding "Annotatedimage" as an alias for "AnnotatedImage" like we do for "Highlightedtext = HighlightedText" at the bottom of this file

Reply:
done